components/public-api-server/pkg/auth/context.go

```diff
@@ -23,7 +23,9 @@ const (
 )
 
 type Token struct {
-	Type  TokenType
+	Type TokenType
+	// When Type is AccessTokenType, the value is the raw token value
+	// When Type is CookieTokenType, the value is cookie_name=cooke_value
```

Comment:
Not really important, but: Di you considered making this Type, Value string, CookieName string (optional)? :thinking: Motivation being that clients don't have to parse Value over and over again. But not sure how it's used, so: leaving it up to you :wink: